### DISCUSSION TITLE
Feature Request: Ability to lock/pin time range selection

### DISCUSSION THREAD

**Discussion by shousper** [grafana/grafana#48608] [Dashboards, Panels & Visualizations]
# Idea

A 🔒 toggle button next to the time range selector, when locked, persists the selected time range when navigating between dashboards or opening new Grafana tabs _from_ the locked one (this prevents changes to the default behaviour).

# Imperative

We often analyse our metrics over specific day (e.g. last Saturday) or by drilling into a particular event (like a log message or anomaly in a chart). Given all our metrics can't (shouldn't) be on 1 dashboard, this often involves opening many Grafana tabs for the various dashboards of interest and then copy/pasting the `from=...&to=...` query parameters between the tabs & reloading them.

If we could instead start from any dashboard that can be used to dial in the time range and click a little 🔒 button to pin the time range when navigating between dashboards or opening new Grafana tabs (from the current one), we could skip the copy/paste.

# Alternatives

I've considered configuring data links to speed this up.. but we have _a lot_ of dashboards, many of which are imported & not maintained by us. A simplistic feature like this would be universal.

Thanks for reading 🙏🏻

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
